public/resources/remix/index.js

```diff
@@ -83,10 +83,15 @@
   }
 
   function injectStyleSheets(metadata) {
-    var stylesheets =
-      "<link href=\"" + metadata.host + "/resources/remix/clean-slate.min.css\" rel=\"stylesheet\">\n" +
-      "<link href=\"" + metadata.host + "/resources/remix/style.css\" rel=\"stylesheet\">\n";
-    document.head.appendChild(stylesheets);
+    var stylesheet1 = document.createElement("link");
+    stylesheet1.rel = "stylesheet";
+    stylesheet1.href = "\"" + metadata.host + "/resources/remix/clean-slate.min.css\"";
```

Comment:
you don't need quotes inside an href, so just `metadata.host + "/resources/remix/clean-slate.min.css"`. The quotes above are only because the HTML code needs quotes in it, and is itself built as a string in JavaScript so already has quotes around it =)